tdrs-backend/gunicorn_start.sh

```diff
@@ -19,10 +19,10 @@ python manage.py migrate
 python manage.py populate_stts
 python manage.py collectstatic --noinput
 
-celery -A tdpservice.settings worker -c 1 --max-memory-per-child 5000 &
+celery -A tdpservice.settings worker -c 1 &
 sleep 5
 # TODO: Uncomment the following line to add flower service when memory limitation is resolved
-# celery -A tdpservice.settings --broker=$REDIS_URI flower &
+celery -A tdpservice.settings --broker=$REDIS_URI flower &
```

Comment:
We should be able to create an endpoint/view in django for monitoring services like: celery and redis that are not part of core web service. and then we can provide information in the admin page